docs/parser.md

```diff
@@ -0,0 +1,502 @@
+# GodTools Parser Documentation
+
+This document describes the XML parser system used in the know-god-web application to render GodTools content.
+
+## Overview
+
+The parser converts GodTools XML manifests into TypeScript/JavaScript objects that can be rendered in the web application.
+
+### Content Creation and Publishing Flow
+
+Admins create and edit GodTools content through the **mobile-content-admin** application, which provides tools to author XML pages based on the defined schema. Once content is created and approved, the XML files are published and made available to client applications.
+
+### Parsing Flow
+
+```
+XML Files (created in mobile-content-admin)
+    ↓
+Kotlin Multiplatform Parser (kotlin-mpp-godtools-tool-parser)
+    ↓
+NPM Package (@cruglobal/godtools-shared)
+    ↓
+This Web Application (know-god-web)
+```
+
+### Package Information
+
+- **NPM Package**: `@cruglobal/godtools-shared`
+- **Source Repository**: [kotlin-mpp-godtools-tool-parser](https://github.com/CruGlobal/kotlin-mpp-godtools-tool-parser)
+- **XML API/Schema**: [mobile-content-admin](https://github.com/CruGlobal/mobile-content-admin)
```

Comment:
we do have the xml schema defined here: https://github.com/CruGlobal/mobile-content-api/tree/master/public/xmlns